src/render-decorator.coffee

```
@@ -23,11 +29,11 @@ class RenderDecorator
 
   # @param {Object} middleware Contains function that consumes Livingdoc object and returns updated Livingdoc object
   usePreRender: (middleware) ->
-    unless middleware.handler? or middleware.scope?
+    unless middleware?.handler? and middleware?.scope?
       return console.error('RenderDecorator#usePreRender: invalid middleware structure, scope and handler keys expected')
```

Please just throw errors her instead of logging something. That's easier to test.